### DISCUSSION TITLE
Can't access Overseerr through external ip anymore

### DISCUSSION THREAD

**Discussion by DaisyAge12** [sct/overseerr#3921] [Help]
I had overseerr configured with nginx proxy manager and duckdns to reroute to my external  ip address so that my family could request movies and TV. The whole set up was working fine for a day but today everything is broken. I get 502 Bad gateway. I removed the proxy and DDNS and still can't access through external. I deleted my overseerr container and started a new one and still can't get anyone else into overseerr. I've got everything port forwarded in the router and the overseerr logs don't show any errors. I'm at a loss, nothing is working and I'm willing to just start from scratch.

**Comment by Lockszmith-GH** [comment]
Hello @DaisyAge12, welcome to the community.
I am not a maintainer here in the repo, so nothing I say is 'official', but I'll be happy to help you troubleshoot this

I'll start with - this isn't an Overseerr issue - if you can access it with the local IP address, it has nothing to do with Overseerr.
With that out of the way.

From what you wrote I gather your using a **docker** based setup (hosted on linux? x86 or ARM?) with **NPM** and **Overseer** on separate containers - right?

Please share how the overseer entry is configured in your NPM

Are you using NPM as a frontend of anything else that works? If so, please share a working entry as well.

Are you using docker-compose? What networking settings are you using for NPM and for the Overseer containers?
